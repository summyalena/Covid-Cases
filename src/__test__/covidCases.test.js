import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';
import CovidCase from '../components/covidCaseHome/covidCase';
import store from '../redux/configureStore';

it('render covid-case', () => {
  const component = renderer.create(
    <Provider store={store}>
      <MemoryRouter>
        <CovidCase />
      </MemoryRouter>
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
