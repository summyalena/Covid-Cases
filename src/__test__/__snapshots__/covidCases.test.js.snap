// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render covid-case 1`] = `
<div
  className="covidCase"
>
  <div
    className="heading"
  >
    <h3>
      Stats of Cases by Countries
    </h3>
  </div>
  <div
    className="container"
  >
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Afghanistan"
          className="img"
          src="afghanistan.png"
        />
        <a
          href="/details:Afghanistan"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Afghanistan
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Albania"
          className="img"
          src="albania.png"
        />
        <a
          href="/details:Albania"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Albania
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Algeria"
          className="img"
          src="algeria.png"
        />
        <a
          href="/details:Algeria"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Algeria
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Andorra"
          className="img"
          src="andorra.png"
        />
        <a
          href="/details:Andorra"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Andorra
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Angola"
          className="img"
          src="angola.png"
        />
        <a
          href="/details:Angola"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Angola
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Antarctica"
          className="img"
          src="antarctica.png"
        />
        <a
          href="/details:Antarctica"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Antarctica
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Antigua and Barbuda"
          className="img"
          src="antigua-and-barbuda.png"
        />
        <a
          href="/details:Antigua and Barbuda"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Antigua and Barbuda
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Argentina"
          className="img"
          src="argentina.png"
        />
        <a
          href="/details:Argentina"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Argentina
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Armenia"
          className="img"
          src="armenia.png"
        />
        <a
          href="/details:Armenia"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Armenia
      </h2>
    </div>
    <div
      className="covidHome"
    >
      <ul
        className="div"
      >
        <img
          alt="Australia"
          className="img"
          src="australia.png"
        />
        <a
          href="/details:Australia"
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-circle-arrow-right font"
            data-icon="circle-arrow-right"
            data-prefix="fas"
            focusable="false"
            onClick={[Function]}
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 256C0 397.4 114.6 512 256 512s256-114.6 256-256S397.4 0 256 0S0 114.6 0 256zM297 385c-9.4 9.4-24.6 9.4-33.9 0s-9.4-24.6 0-33.9l71-71L120 280c-13.3 0-24-10.7-24-24s10.7-24 24-24l214.1 0-71-71c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0L409 239c9.4 9.4 9.4 24.6 0 33.9L297 385z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </a>
      </ul>
      <h2>
        Australia
      </h2>
    </div>
  </div>
</div>
`;
